src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Generated/Face/IPersonGroupOperations.cs

```diff
@@ -18,23 +18,23 @@ namespace Microsoft.Azure.CognitiveServices.Vision.Face
     using System.Threading.Tasks;
 
     /// <summary>
-    /// PersonGroup operations.
+    /// PersonGroupOperations operations.
```

Comment:
Is this comments changeable? Just too wired to have duplicate operations.
If it is not easy achievable, please ignore this issue.

Reply:
It's because we have object called PersonGroup, no easy naming solution to change this.

Reply:
resolved.